styles.css

```diff
@@ -610,3 +660,491 @@ li {
 #button_contact:active {
   background-color: #008552;
 }
+
+@media all and (min-width: 768px) {
+  .header2 {
+    display: none;
+  }
+
+  ul {
+    list-style: none;
+  }
+
+  a {
+    text-decoration: none;
+  }
+
+  body {
+    height: 3846px;
+    background-color: #fff;
+    background-image: url(/img/imgdes/img_form1.png), url(/img/imgdes/img_fom2.png), url(/img/imgdes/img_form3.png);
+    background-repeat: no-repeat, no-repeat, no-repeat;
+    background-size: 250px 360px, 130px 106px, 223px 495px;
+    background-position: 0 3650px, 280px 3800px, 1220px 3510px;
+  }
+
+  #heading {
+    background-image: url(/img/imgdes/background_header.png);
```

Comment:
- [ ] As the previous reviewer has mentioned that your background image is not showing up. it is not fixed yet. I have checked and found out that the root of the problem is that you need to add a dot in the beginning of the link in the background URL like this : `url(./img/imgdes/background_header.png)`. doing this will fix the problem.
- [ ] Also the same case is with the down arrow icon. kindly consider fixing it with the same method I described.
-  these images may have shown in your local machine but to make them work in proper browser environment, our links should contain complete link.